src/main/kotlin/solve/importer/view/ButtonView.kt

```diff
@@ -0,0 +1,85 @@
+package solve.importer.view
+
+import javafx.geometry.Insets
+import javafx.scene.control.Button
+import solve.importer.ProjectParser.createAlert
+import solve.main.MainController
+import solve.menubar.view.MenuBarView
+import solve.project.model.ProjectFrame
+import tornadofx.*
+
+
+class ButtonView : View() {
+
+    private val mainController: MainController by inject()
+
+    private val directoryPathView: DirectoryPathView by inject()
+
+    private fun cancelAction() {
+        MenuBarView().importer.close()
+    }
+
+    private fun importAction(button: Button) {
+        val projectVal = directoryPathView.project.value
+        try {
+            mainController.sceneFacade.visualize(projectVal.layers, projectVal.frames.map { it.frame })
```

Comment:
Let's make ``sceneFacade`` field private and pass data into the scene through ``MainController``. Importer and other components should not depend on ``sceneFacade``